src/components/MainTabs/index.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import BugzillaComponents from '../BugzillaComponents';
+import Reportees from '../Reportees';
+
+function TabContainer(props) {
+  const { children } = props;
+  return (
+    <Typography component="div" style={{ padding: 8 * 3 }}>
+      {children}
+    </Typography>
+  );
+}
+
+TabContainer.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: theme.palette.background.paper,
+  },
+});
+
+class MainTabs extends React.Component {
+    state = {
+      value: 0,
+    };
+
+    handleChange = (event, value) => {
+      this.setState({ value });
```

Comment:
Use `selectedTabIndex` as the variable name since it is a more meaningful name.

Reply:
Thanks I updated.